projects/go-lib/src/lib/components/go-button/go-button.component.spec.ts

```diff
@@ -105,6 +105,37 @@ describe('GoButtonComponent', () => {
       expect(component.classObject['go-button--dark']).toBe(true);
       expect(component.classObject['go-button--loading']).toBe(true);
     });
+
+    it('return an object that sets go-button--icon-only to false if buttonIcon is set and there is content in the button', () => {
```

Comment:
I probably should have made it stand out more in my previous comment on this, but I was thinking an 's' should be added to 'return' as well.